add-ons/fabric8-launcher/fabric8-launcher.addon

```diff
@@ -0,0 +1,20 @@
+###################################################################################################
+# Name: fabric8-launcher
+# Description: Installs Fabric8 Launcher in a project named rhoarpad 
+# Required-Vars: GITHUB_USERNAME, GITHUB_TOKEN 
+# OpenShift-Version: >3.9.0       
+# Var-Defaults: OPENSHIFT_USERNAME=developer,OPENSHIFT_PASSWORD=developer,CONSOLE_URL=https://#{ip}:8443
```

Comment:
@VeerMuchandi you should remove `OPENSHIFT_USERNAME=developer,OPENSHIFT_PASSWORD=developer` this is not necessary.